cephfs/cephfs.go

```diff
@@ -70,6 +70,22 @@ func (mount *MountInfo) ReadDefaultConfigFile() error {
 	return getError(ret)
 }
 
+// ParseConfigArgv configures the mount using a unix style command line
+// argument vector.
+//
+// Implements:
+//  int ceph_conf_parse_argv(struct ceph_mount_info *cmount, int argc, const char **argv);
+func (mount *MountInfo) ParseConfigArgv(argv []string) error {
+	cargv := make([]*C.char, len(argv))
+	for i := range argv {
+		cargv[i] = C.CString(argv[i])
+		defer C.free(unsafe.Pointer(cargv[i]))
+	}
+
```

Comment:
Shoudl we check for  mount.mount == nil?

Reply:
yes, and maybe verify that `argv` is not empty/nil?

Reply:
Sure. I'll check for how the ceph call reacts to argv edge conditions and if it is inelegant I'll add an additional check on our side.